ui/widgets/nuxeo-selectivity.js

```diff
@@ -6953,7 +6951,7 @@ input[type='text'].selectivity-multiple-input:focus {
 
 #input:not([readonly]) .selectivity-single-select {
   border-bottom: 1px solid #3a3a54;
-  border-radius: 2px;
+  min-height: calc(2em + 4px);
```

Comment:
Isn't this reductant given that we already have the same rule on `.selectivity-single-select`? 🤔